akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -76,7 +76,7 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem)
       }
       val tls = TLS(createEngine, _ ⇒ Success(()), IgnoreComplete)
 
-      AlpnSwitch(getChosenProtocol, Http().serverLayer(), http2Layer()) atop
+      AlpnSwitch(getChosenProtocol, http.serverLayer(), http2Layer()) atop
```

Comment:
Wasn't the @jrudolph comment here https://github.com/akka/akka-http/issues/1160#issue-231293957 about passing settings to `serverLayer` like so:

```scala
serverLayer(settings = settings)
```